crates/rnote-ui/src/appwindow/actions.rs

```diff
@@ -761,6 +769,7 @@ impl RnAppWindow {
         app.set_accels_for_action("win.keyboard-shortcuts", &["<Ctrl>question"]);
         app.set_accels_for_action("win.open-canvasmenu", &["F9"]);
         app.set_accels_for_action("win.open-appmenu", &["F10"]);
+        app.set_accels_for_action("win.toggle-overview", &["F12"]);
```

Comment:
Should be discussed.

I saw other application mapping this to `Ctrl+O`, but in our case this shortcut is already used for `win.open-doc`.

Reply:
what about `Ctrl+Shift+o` ?

Reply:
I think its too long for a **toggle** shortcut.

I saw <kbd>Ctrl+Shift+o</kbd> being used in `Gnome Console`, but their decision was limited by the fact that it was a terminal.

Reply:
I am leaning towards `Ctrl+Shift+O`, following convention